exercises/ex1.cpp

```diff
@@ -11,38 +9,39 @@ int main()
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
 
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
+switch (week) {
+case 1:
+   cout << "Monday" << endl;
+   break;
+
+case 2:
+   cout << "Tuesday" << endl;
+   break;
+
+case 3:
+   cout << "Wednesday" << endl;
+   break;
+
+case 4:
+   cout << "Thursday" << endl;
+   break;
+
+case 5:
+   cout << "Friday" << endl;
+   break;
+
+case 6:
     cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
+   break;
+
+case 7:
     cout << "Sunday" << endl;
```

Comment:
endl è ridondante, potresti scriverlo dopo lo switch case e rimuoverlo da tutti i cases
Inoltre, potresti spostare l'implementazione dello switch case in una funzioone a parte in modo da non riempire troppo la funzione main